billing/unified-invoices.md

```diff
@@ -17,38 +17,72 @@ Each month, for each organization, a single invoice is generated, including the
 
 ## Monthly invoice
 
-At the begin of every month, a single new invoice is created. It provisions your account for the month to come:
+At the beginning of each month, a new invoice is generated for each organisation. It details the consumption of services, and the provisioning of your account for the coming month.
```

Comment:
It feels like we more often use `organization` (with a `z`) in the other documents rather than `organisation`.

Reply:
Noted!